client/me/sidebar/style.scss

```diff
@@ -14,3 +14,42 @@
 .is-group-me .sidebar__menu-happychat {
 	display: none;
 }
+
+.sidebar {
+	.sidebar__menu-icon {
+		height: 20px;
+		width: 20px;
+	}
+	.sidebar__me-signout-button {
+		.gridicons-popout {
+			display: none;
+		}
+	}
+}
+
+.is-sidebar-collapsed .sidebar {
+	.gravatar {
+		width: 20px;
+		height: 20px;
+		border: 2px solid var( --color-border-inverted );
+		position: relative;
+		left: 1px;
+	}
+	.sidebar__me-signout-button {
+		padding: 3px;
+		margin-top: 15px;
+		span {
```

Comment:
I personally don't like using `span` or `div` in CSS selectors because they're basically semantic-less. `<span>` is the right element to use here since but I'd add a class name to it.

That might just be personal opinion though.

Reply:
I haven't used it in a project yet, but I'd really like to! It wouldn't have to be Tailwind specifically, I'd just really like to try some of those atomic CSS ideas to see what they're like to use in practice. I really like some of the ideas behind it. 👍